test/MarketMaker.js

```diff
@@ -162,6 +215,69 @@ contract('marketMaker', function(accounts) {
     await help.waitToBlock(startBlock+blocksPerPeriod*numberOfMonth, accounts);
   };
 
+  it("Should return the correct sellPrice on every period", async function() {
+    token = await simulateCrowdsale(100, [40,30,20,10,0], accounts);
+
+    const startBlock = web3.eth.blockNumber+10;
+    const blocksPerPeriod = 5;
+    const initialPriceSpread = 100500;
+
+    mm = await LifMarketMaker.new(
+      token.address, startBlock, blocksPerPeriod, 24,
+      accounts[1], initialPriceSpread,
+      {value: web3.toWei(8, 'ether'), from: accounts[0]}
+    );
+
+    const initialSellPrice = parseInt(await mm.initialSellPrice());
```

Comment:
we should not use a number extracted from the contract as the basis for our expected results. I mean, in some scenarios that's valid, but not here, because this is exactly what we are testing: the price